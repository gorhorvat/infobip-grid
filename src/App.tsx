import React from 'react';
import GridContainer from './containers/grid-container/grid-container';

function App() {
  return <GridContainer />;
}

export default App;
