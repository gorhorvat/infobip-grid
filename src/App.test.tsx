import React from "react";
import { shallow } from 'enzyme';
import App from "./App";

describe('App', () => {
  function setup() {
    return shallow(<App />);
  }

  it('Renders without crashing', () => {
    // Arrange
    const wrapper = setup();

    // Assert
    expect(wrapper).toMatchSnapshot();
  });
});