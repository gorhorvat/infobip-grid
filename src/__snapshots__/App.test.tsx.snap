// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders without crashing 1`] = `ShallowWrapper {}`;
